make/config.mk

```diff
@@ -214,6 +214,10 @@ EXTRA_OPERATORS =
 # Create C++ interface package
 USE_CPP_PACKAGE = 0
 
+# Use int64_t type for tensor size.
+# This will cause performance degradation reported in issue #14496
```

Comment:
Maybe add something to say when to use it too, like "Set to 1 for large tensor sizes, when tensors will have large shapes such as for graphs" or something...?

Reply:
updated.